src/app/main/component/shared/components/events-list-item/events-list-item.component.spec.ts

```diff
@@ -419,22 +418,6 @@ describe('EventsListItemComponent', () => {
       component.checkButtonStatus();
       expect(component.btnStyle).toEqual(component.styleBtn.hiden);
     });
-
-    it('should set btnStyle and nameBtn correctly when user is unsubscribed and event is active', () => {
```

Comment:
Is it possible to save this test?

Reply:
This test checks styles and button names based on the value of the canUserJoinCloseEvent property. However, this property has been removed, so the test no longer makes sense.